sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Testing/EventDataExtensions.cs

```diff
@@ -65,7 +65,7 @@ public static bool IsEquivalentTo(this EventData instance,
             // Verify that the stand-alone system properties are equivalent.
 
             if ((considerSystemProperties)
-                && ((instance.Offset != other.Offset)
```

Comment:
We know that `Offset` is an alias for `OffsetString`, so the value can't differ.   We also can't safely call `Offset` as it will throw in the modern offset string format.   I think we're good just comparing the string value here, unless you're thinking of something that I'm missing.